packages/sample-app/webpack.config.ts

```diff
@@ -0,0 +1,155 @@
+import path from 'path';
+import CopyPlugin from 'copy-webpack-plugin';
+import CSSMinimizerPlugin from 'css-minimizer-webpack-plugin';
+import HTMLPlugin from 'html-webpack-plugin';
+import _ from 'lodash';
+import MiniCSSExtractPlugin from 'mini-css-extract-plugin';
+import type { Configuration, WebpackPluginInstance } from 'webpack';
+import { EnvironmentPlugin, container } from 'webpack';
+import { getSampleAppSharedModules } from './src/plugin-sdk-addons/shared-modules';
+
+const isProd = process.env.NODE_ENV === 'production';
+
+const pathTo = (relativePath: string) => path.resolve(__dirname, relativePath);
+
+const sharedModules = getSampleAppSharedModules({
+  eager: true,
+  requiredVersion: false,
```

Comment:
This is only necessary because we have

```json
"@openshift/dynamic-plugin-sdk": "portal:../lib-core",
```

in `packages/sample-app/package.json` so webpack `ModuleFederationPlugin` thinks that `portal:../lib-core` is actually a semver range when it's not.

At runtime, webpack will complain that sample app is consuming `@openshift/dynamic-plugin-sdk` shared module but the expected vs. actual version is not matching.

I'll modify this code to manually resolve `requiredVersion` specifically for `@openshift/dynamic-plugin-sdk` shared module.